docs/sources/writing-guide/documentation-structure/index.md

```diff
@@ -54,6 +54,26 @@ For more information about how to write tasks, refer to [Tasks]({{< relref "../t
 
 > **Note:** If a directory contains multiple pages or subdirectories, it is a branch bundle, and it must include an `_index.md` file. A directory containing only one page is a leaf bundle, and the content filename must be `index.md`.
 
-## Useful links
+## Pages and page bundles
 
-- For more information about branch bundles and leaf bundles, refer to [Page bundles](https://gohugo.io/content-management/page-bundles/).
+Each web page generated by Hugo comes from one of three source files:
+- `page/_index.md`: a Hugo branch bundle
+- `page/index.md`: a Hugo leaf bundle
+- `page.md`: a Hugo page
+
+Although each of the preceding examples results in the same URL (`/page/`), Hugo works with each source file differently.
+
+**Branch bundles** (`page/_index.md`) are required to produce page hierarchies.
+For the `/page/subpage/` URL to generated, there must first be a `page/_index.md` branch bundle source file.
+
+**Leaf bundles** (`page/index.md`) are required to conveniently and consistently bundle page assets.
```

Comment:
This is utterly confusing to me.  As a TW I shouldn't be thinking about the CSS, that should just be part of the template.  And I don't think we want community members/developers changing the CSS.

Reply:
I think we might be safer just going with the text from the Hugo page you linked to and saying "A leaf means it has no children."

Reply:
Although the Hugo documentation describes what a leaf bundle is, it doesn't explain why one might use it. I can think of an alternative to `CSS` for bundling purpose, perhaps a static file for download makes more sense?